### ISSUE TITLE
frequently asked questions

### ISSUE THREAD

**Issue by ahdinosaur** [ahdinosaur/inu#28]
- [ ] how to  dispatch effects? (wrap in a run action)
- [ ] [help, my view dispatch is causing an infinite loop!](https://github.com/ahdinosaur/inu/pull/26#issuecomment-234424389) (wrap in a `process.nextTick`)
- [ ] how does this compare with X?
  - [ ] react + redux + redux-loop
  - [ ] choo
  - [ ] elm
  - [ ] cyclejs
- [ ] how to add logging ([`inu-log`](https://github.com/ahdinosaur/inu-log))
- [ ] how to use compose apps with `tcomb`
- [ ] how to use compose apps with `inux`

anything else?

**Comment by gcanti** [comment]
Hi @ahdinosaur,

what you mean by "how to use compose apps with `tcomb`"? Maybe I can help

**Comment by ahdinosaur** [comment]
@gcanti based on [your documentation for using `tcomb` in `tom`](https://github.com/gcanti/tom#reducing-the-boilerplate-and-adding-type-safety), @pietgeursen and i have used `tcomb` with `inu` in [`ssb-gathering-ui`](https://github.com/pietgeursen/ssb-gathering-ui) and [`inu-plays-roguelike`](https://github.com/ahdinosaur/inu-plays-roguelike), respectively. the idea was to document this as we've done. maybe not "compose apps with `tcomb`" but rather "use `tcomb` to make an app". :smile_cat:

**Comment by crazy4groovy** [comment]
Is there any virtual dom in this (given the tagged template literal syntax)? I know it's kind of a buzzword now but still good to know upon browsing, when evaluating frameworks.

**Comment by ahdinosaur** [comment]
@crazy4groovy yep, `yo-yo` allows us to create html elements with the template literal syntax (using `bel`) and update html elements with diffs (using `morphdom`). so it's a very similar developer experience as with virtual dom implementations, but using the real dom.
